tasks/br/data.py

```diff
@@ -70,15 +69,17 @@ def _get_filename(self):
         cmd += ' | '
         cmd += 'awk \'{print $9}\''
         cmd += ' | '
-        cmd += 'grep -i {state}_[0-9].*zip$'.format(state=self.state)
+        cmd += 'grep {state}_[A-Za-z_]*[0-9].*zip$'.format(state=self.state.upper())
 
         return shell(cmd)
 
     def download(self):
-        filename = self._get_filename()
-        shell('wget -O {output}.zip {url}{filename}'.format(
-            output=self.output().path, url=self.URL, filename=filename
-        ))
+        filenames = self._get_filename().splitlines()
+
+        for filename in filenames:
+            shell('wget -O {output}.zip {url}{filename}'.format(
+                output=self.output().path, url=self.URL, filename=filename
```

Comment:
🤔 This is going to download two times into the same output file, is it no?

Reply:
It's iterating over the output of `curl | awk | grep` to extract each line file name. For example the output of 
```
curl -s ftp://ftp.ibge.gov.br/Censos/Censo_Demografico_2010/Resultados_do_Universo/Agregados_por_Setores_Censitarios/ | awk '{print $9}' | grep SP_[A-Za-z_]*[0-9].*zip$
```
returns two lines 

```
SP_Capital_20150728.zip
SP_Exceto_a_Capital_20150527.zip
```
And the output of 
```
curl -s ftp://ftp.ibge.gov.br/Censos/Censo_Demografico_2010/Resultados_do_Universo/Agregados_por_Setores_Censitarios/ | awk '{print $9}' | grep ES_[A-Za-z_]*[0-9].*zip$
```

returns one line

```
ES_20161202.zip
```

Reply:
Yep, so the `wget` will run twice, writing to the same file (`output.zip`). So we will download `SP_Capital_20150728.zip`  first, and then overwrite it with `SP_Exceto_a_Capital_20150527.zip`.

it appears we would be losing the data from the capital file.